environment.yml

```
@@ -0,0 +1,35 @@
+#name: z
+channels:
+  - defaults
+  - ome
+  - conda-forge
+  - anaconda
+dependencies:
+  - openjdk
+  - maven
+  - make
+  - z5py
+  - python == 3.7.9
+  - pyside2
+  - napari
+  - flake8
+  - ipython
+  - mypy
+  - omero-py
+  - opencv
+  - pip
+  - py-opencv
+  - pytest
+  - requests
+  - s3fs
+  - scikit-image
+  - scipy
+  - xarray
+  - zarr >= 2.4.0
+  - pip:
+      - pre-commit
+      - pytest-qt
```

`conda-forge` packages of both of these exist. That said, do we need `pre-commit`? Is this environment intended for development or just testing?

```suggestion
  - pre-commit
  - pytest-qt
```

---

Sorry. This is a copy of the ome-zarr-py dev environment. Thanks for catching it.